### PR TITLE
test: Update fedora-atomic to use Fedora 24 base

### PR DESCRIPTION
**Pull request by stefwalter**

### CODE REVIEW

**Comment by stefwalter** [comment]
Uploading....

**Comment by stefwalter** [comment]
Network teaming doesn't work on Fedora Atomic 24.

We see: "Master connection: System eth0 can't be activated: No device available"

**Comment by stefwalter** [comment]
testnetworking-testteam-fail

**Comment by stefwalter** [comment]
The Team stuff is still broken on fedora-atomic 24

**Comment by mvollmer** [comment]
> We see: "Master connection: System eth0 can't be activated: No device available"

That's a misleading error message, unfortunately.  The actual reason is that the team plugin is not installed.

The error message appears when a slave of a team is activated, without the team plugin.  With the plugin, the activation succeeds.  I'll file a bug about improving the wording.

**Comment by mvollmer** [comment]
>  I'll file a bug about improving the wording.

https://bugzilla.redhat.com/show_bug.cgi?id=1375705

**Comment by mvollmer** [comment]
> The Team stuff is still broken on fedora-atomic 24

The team stuff was always broken on the Atomics but we managed to fly under the radar in the past.  #5045 should resolve this.

**Comment by dperpeet** [comment]
I think the commit message in https://github.com/cockpit-project/cockpit/commit/e577cb6fb48809f7fc1a5a5cfaed751994e12d41 was wrong.

**Comment by dperpeet** [comment]
unblocked, testing again

File: test/images/scripts/rhel-atomic.bootstrap [hunk 1]
```diff
@@ -4,7 +4,7 @@
 set -e
 
 url="http://cdn.stage.redhat.com/content/dist/rhel/atomic/7/7Server/x86_64/images/"
-prefix="rhel-atomic-cloud-7.2"
+prefix="rhel-atomic-cloud-7.2-[0-9\.]+.x86_64.qcow2"
```
Comment (dperpeet) [inline test/images/scripts/rhel-atomic.bootstrap]: I think this rhel atomic change would be better placed in its own pull request - together with an image update.
We'll probably want to switch to 7.3 soon (or now?).

**Comment by dperpeet** [comment]
adding priority, because we want to have newer images for our atomic images to get the tests current again

File: test/images/scripts/rhel-atomic.bootstrap (hunk 1 above)
Comment (dperpeet) [inline test/images/scripts/rhel-atomic.bootstrap]: Also, this naming pattern seems to be outdated. Starting in June, the names look like this: `rhel-atomic-cloud-7.2.5.x86_64.qcow2` and `rhel-atomic-cloud-7.2.6-1.x86_64.qcow2`

**Comment by dperpeet** [comment]
I will post a pull request with fixes

**Comment by dperpeet** [comment]
replaced by #5152
